pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -382,5 +382,5 @@ func manageOpenShiftControllerManagerDeployment_v311_00_to_latest(client appscli
 		}
 	}
 
-	return resourceapply.ApplyDaemonSet(client, recorder, required, resourcemerge.ExpectedDaemonSetGeneration(required, generationStatus), forceRollout)
+	return resourceapply.ApplyDaemonSet(client, recorder, required, resourcemerge.ExpectedDaemonSetGeneration(required, generationStatus))
```

Comment:
What is the impact of not having the `forceRollout` option? I'm pretty sure we heavily relied on this in the operator.

Reply:
Fixed, looks like the ApplyDaemonsetForce is going away in a bit so we'll have to fix that at some point also.

Reply:
Personally I'd like to do a major refactor of this operator. `library-go` has a lot of canned controllers that are shared across multiple CVO operators that we should take advantage of.

Reply:
Tracking the refactor in https://issues.redhat.com/browse/BUILD-113